src/main/java/org/saltyrtc/client/signaling/Signaling.java

```diff
@@ -466,28 +466,28 @@ public void onDisconnected(WebSocket websocket,
                             getLogger().warn("WebSocket closed, server is being shut down");
                             break;
                         case CloseCode.NO_SHARED_SUBPROTOCOL:
-                            getLogger().error("No shared sub-protocol could be found");
+                            getLogger().error("WebSocket closed: No shared sub-protocol could be found");
                             break;
                         case CloseCode.PATH_FULL:
-                            getLogger().error("Path full (no free responder byte)");
+                            getLogger().error("WebSocket closed: Path full (no free responder byte)");
                             break;
                         case CloseCode.PROTOCOL_ERROR:
-                            getLogger().error("Protocol error");
+                            getLogger().error("WebSocket closed: Protocol error");
                             break;
                         case CloseCode.INTERNAL_ERROR:
-                            getLogger().error("Internal server error");
+                            getLogger().error("WebSocket closed: Internal server error");
                             break;
                         case CloseCode.DROPPED_BY_INITIATOR:
-                            getLogger().warn("Dropped by initiator");
+                            getLogger().warn("WebSocket closed: Dropped by initiator");
                             break;
                         case CloseCode.INITIATOR_COULD_NOT_DECRYPT:
-                            getLogger().error("Initiator could not decrypt message");
+                            getLogger().error("WebSocket closed: Initiator could not decrypt message");
```

Comment:
`.warn`?

Reply:
Well, this condition indicates a bug somewhere, so I'd leave it at error. If you disagree, why?

Reply:
After discussion: From SaltyRTC perspective it's not an error, only from application perspective.